enum_var.go

```diff
@@ -27,7 +27,10 @@ type EnumVar struct {
 }
 
 func (e *EnumVar) String() string {
-	return *e.value
+	if e != nil && e.value != nil {
```

Comment:
I think `e != nil` is redundant since if it were nil, the exception would happen before calling `.String()`